src/App.jsx

```diff
@@ -137,13 +139,35 @@ function KanbanBoard() {
   const handleTaskUpdate = async (taskId, updates) => {
     if (!selectedTask?.columnId) return;
     await updateTask(selectedTask.columnId, taskId, updates);
+    setSelectedTask(prev => ({
+      ...prev,
+      ...updates
+    }));
   };
 
   const handleTaskDelete = async (taskId) => {
     if (!selectedTask?.columnId) return;
     await deleteTask(selectedTask.columnId, taskId);
   };
 
+  const handleAddTask = async (columnId) => {
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```suggestion
const handleAddTask = async (columnId) => {
  if (!newTaskContent?.trim()) return;
  try {
    await addTask(columnId, {
      id: crypto.randomUUID(),
      content: newTaskContent.trim(),
      createdAt: dueDate.toISOString(),
      dueDate: dueDate.toISOString(),
      labels: []
    });
    setNewTaskContent('');
    setAddingTaskToColumn(null);
  } catch (error) {
    console.error('Failed to add task:', error);
    // Handle error appropriately (e.g., show user feedback)
  }
};
```
The handleAddTask function should handle potential errors from the addTask operation to prevent silent failures and ensure proper user feedback.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;